build_daemon/lib/data/build_target.dart

```diff
@@ -4,6 +4,7 @@ import 'package:built_value/serializer.dart';
 
 part 'build_target.g.dart';
 
+/// The build target, e.g. folder.
```

Comment:
Add a note that what a target means is specific to the builder implementation/build system. It is a transparent identifier as far as the protocol is concerned.